### PR TITLE
Fix #2038: File formatting under numericinput package

### LINKED ISSUE #2038: Fix file formatting under numericinput package
**Describe the bug**
All the files, under the package `domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput` are not in the correct format. 

**Expected behavior**
File format should be follow from this file - [TextInputEqualsRuleClassifierProviderTest](https://github.com/oppia/oppia-android/blob/develop/domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProviderTest.kt
)

**Correct Order**
Constant values
Dagger injection
Local variables
Test Before setup
Tests
Dagger Setup

### PR DESCRIPTION
**Pull request by Victor-Titan**
## Explanation

Fixes #2038 Formatted the files under the numericinput package as specified in issue #2038 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

PTAL @anandwana001

### CODE REVIEW

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Remove the function from the test file as we are accessing them from `InteractionObjectTestBuilder ` and they are not in use any more.

File: domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt [hunk 1]
```diff
@@ -67,13 +77,13 @@ class NumericInputEqualsRuleClassifierProviderTest {
   @Test
   fun testPositiveRealAnswer_positiveRealInput_valuesInRange_bothValuesMatch() {
     val inputs = mapOf(
-      "x" to createReal(
+      "x" to InteractionObjectTestBuilder.createReal(
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: We should use variables like above for this one too.

Comment (Victor-Titan) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: Can you elaborate please?

Comment (anandwana001) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: Above where we are creating variable like
```kotlin
  private val POSITIVE_REAL_VALUE_3_5 = createReal(value = 3.5)	    InteractionObjectTestBuilder.createReal(value = 1.5)
```
create and use variables here. not directly calling the method and getting the value.

**Comment by anandwana001** [comment]
@ARJUPTA Would you help @Victor-Titan here as you are also working on the same type of issues, it will be helpful.

**Comment by Arjupta** [comment]
> @ARJUPTA Would you help @Victor-Titan here as you are also working on the same type of issues, it will be helpful.
Sure @anandwana001

File: domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt [hunk 2]
```diff
@@ -67,13 +77,13 @@ class NumericInputEqualsRuleClassifierProviderTest {
...
         value = 5 * FLOAT_EQUALITY_INTERVAL
       )
     )
 
     val matches = inputEqualsRuleClassifier.matches(
-      answer = createReal(value = 5 * FLOAT_EQUALITY_INTERVAL + FLOAT_EQUALITY_INTERVAL / 10),
+      answer = InteractionObjectTestBuilder.createReal(value = 5 * FLOAT_EQUALITY_INTERVAL + FLOAT_EQUALITY_INTERVAL / 10),
```
Comment (Arjupta) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: This line has exceeded max line length (100) . For avoiding this kind of issue, try to keep your code behind the grey line that is shown in Android Studio. Refer to KTlint Wiki for more details

**Review by Arjupta — CHANGES_REQUESTED** [review]
@Victor-Titan check the comment

**Comment by rt4914** [comment]
@Victor-Titan Is this PR ready for review? If so please mention something like this: `@rt4914 @anandwana001 Please take a look`.
Also, please reply to all the comments made by reviewer.

**Comment by Victor-Titan** [comment]
> @Victor-Titan Is this PR ready for review? If so please mention something like this: `@rt4914 @anandwana001 Please take a look`.
> Also, please reply to all the comments made by reviewer.

Hi! My pushed code seems to keep having errors in the Lint tests when in fact it works perfectly and passes the ktlint checks I'm doing.

**Comment by anandwana001** [comment]
> > @Victor-Titan Is this PR ready for review? If so please mention something like this: `@rt4914 @anandwana001 Please take a look`.
> > Also, please reply to all the comments made by reviewer.
> 
> Hi! My pushed code seems to keep having errors in the Lint tests when in fact it works perfectly and passes the ktlint checks I'm doing.

You can also check ktlint on your local machine to verify and fix errors. Check the wiki guide here - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide

**Comment by Arjupta** [comment]
@Victor-Titan you have some unused imports in some files. Try to find and remove them

**Comment by anandwana001** [comment]
@Victor-Titan any update on this?

**Comment by Victor-Titan** [comment]
> @Victor-Titan any update on this?

@anandwana001 I rectified the ktlint error but Roboelectric tests are showing a build fail whereas the app is building succesfully on my machine

**Comment by Arjupta** [comment]
> > @Victor-Titan any update on this?
> 
> @anandwana001 I rectified the ktlint error but Roboelectric tests are showing a build fail whereas the app is building succesfully on my machine

@Victor-Titan have you checked the logs. They say that there is some issue with the InteractionObject you are using.

### Logs:
 `e: /home/runner/work/oppia-android/oppia-android/domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsLessThanOrEqualToRuleClassifierProviderTest.kt: (233, 38): Unresolved reference: InteractionObject

e: /home/runner/work/oppia-android/oppia-android/domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsLessThanOrEqualToRuleClassifierProviderTest.kt: (234, 12): Unresolved reference: InteractionObject`

**Comment by Victor-Titan** [comment]
@anandwana001 PTAL

File: domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsInclusivelyBetweenRuleClassifierProviderTest.kt [hunk 3]
```diff
@@ -24,20 +24,46 @@ import kotlin.test.fail
...
+
+  private val POSITIVE_INT_VALUE_1 =
+    InteractionObjectTestBuilder.createInt(value = 1)
+
+  private val POSITIVE_INT_VALUE_2 =
+    InteractionObjectTestBuilder.createInt(value = 2)
+
+  private val POSITIVE_INT_VALUE_3 =
+    InteractionObjectTestBuilder.createInt(value = 3)
+  private val NEGATIVE_INT_VALUE_1 =
```
Comment (Arjupta) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsInclusivelyBetweenRuleClassifierProviderTest.kt:59]: Include a space between these lines.

**Review by Arjupta — COMMENTED** [review]
Just a small change

File: domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt (hunk 1 above)
Comment (anandwana001) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: @Victor-Titan Please use variables for this. 

```kotlin
 val inputs = mapOf(
      "x" to createReal(
        value = 5 * FLOAT_EQUALITY_INTERVAL
      )
    )
```

```kotlin
 private val FIVE_TIMES_FLOAT_EQUALITY_INTERVAL =
    InteractionObjectTestBuilder.createReal(value = 5 * FLOAT_EQUALITY_INTERVAL)

 val inputs = mapOf(
      "x" to FIVE_TIMES_FLOAT_EQUALITY_INTERVAL
    )
```

and Remove the below functions from `NumericInputEqualsRuleClassifierProviderTest` as we are going to use all the helper functions from the `InteractionObjectTestBuilder`. 

```kotlin
 private fun createReal(value: Double): InteractionObject {
    return InteractionObject.newBuilder().setReal(value).build()
  }

  private fun createString(value: String): InteractionObject {
    return InteractionObject.newBuilder().setNormalizedString(value).build()
  }
```


Ditto to all other files

Comment (Victor-Titan) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: The test files have changed quite a bit, but there's still places where strings are directly used in the parameters, would you like me to create variables for them and use those instead?

Comment (anandwana001) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: You can first update this branch with the latest develop and continue the work. As per your message in Gitter it looks like you branch is messed up and you are not able to update with the latest develop, in that case I suggest to create a new pr with  fresh branch.

Comment (Victor-Titan) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt]: You are right, I seem to have made a mistake, I checked the main branch code and it hasn't changed the way it has for my local branch. Will rectify and do the needful.

**Comment by anandwana001** [comment]
@Victor-Titan any update on this?

**Comment by anandwana001** [comment]
closing this pr as no further update.

**Pull request closed without merging**